### PR TITLE
increase resources

### PR DESCRIPTION
**Pull request by MajdT51**
MIW and Revocation service require more memory
image

Acapy instances caused high CPU Throttling

### CODE REVIEW

**Comment by matthiashub-porsche** [comment]
LGTM let's try with these numbers and see if anything changes performance wise

File: charts/managed-identity-wallets/templates/deployment.yaml [hunk 1]
```diff
@@ -130,11 +130,11 @@ spec:
           value: {{ .Values.revocation.refreshHour | quote }}
         resources:
           requests:
-            cpu: 100m
-            memory: 128Mi
+            cpu: 2
```
Comment (matthiashub-porsche) [inline charts/managed-identity-wallets/templates/deployment.yaml:133]: Could you only request max 1 CPU? I believe the linter won't run through.

Comment (MajdT51) [reply charts/managed-identity-wallets/templates/deployment.yaml]: request 1 , limit 1 or can limit be 2?

**Review by jpbu — APPROVED** [review]
Looks good to me, values for minimum CPU seem a bit high (adding up to 4 cores for the whole pod)

**Pull request merged**